common/static/common/js/components/views/tabbed_view.js

```diff
@@ -14,6 +14,17 @@
                tabTemplate,
                tabPanelTemplate
            ) {
+                var keys = {
+                    'left':     37,
```

Comment:
You can use the UI Toolkit's constants for these:

http://ui-toolkit.edx.org/utilities/constants/